.github/renovate.json5

```diff
@@ -38,6 +38,9 @@
   gomod: {
     fileMatch: ["go\\.mod\\.tpl$"],
   },
+  dockerfile: {
+    fileMatch: ["Dockerfile\\.goreleaser$", "Dockerfile\\.ubuntu\\.goreleaser$"],
```

Comment:
This means the renovate will be handling upgrading both aline and ubuntu right?

Reply:
> This means the renovate will be handling upgrading both aline and ubuntu right?

💯 Yes